modin/core/storage_formats/base/query_compiler.py

```diff
@@ -50,6 +52,10 @@
 
 from . import doc_utils
 
+if TYPE_CHECKING:
+    # TODO: should be ModinDataframe
```

Comment:
When is this going to be fixed?

Reply:
https://github.com/modin-project/modin/issues/7244